pwiz_tools/Shared/Common/SystemUtil/ProcessRunner.cs

```diff
@@ -147,7 +147,7 @@ public class ProcessRunner : IProcessRunner
                             if (double.TryParse(percentPart.Substring(0, percentPart.Length - 1), out percent))
                             {
                                 percentLast = (int)percent;
-                                status = status.ChangePercentComplete(percentLast);
+                                status = status.ChangePercentComplete(percentLast).ChangeMessage(percentPart);

```

Comment:
This "ChangeMessage(percentPart)" looks suspicious. Isn't that going to just be a number (it's everything between the last space and the percent sign). Wouldn't it be better to leave the message unchanged?

Reply:
Yeah, there's a bug in how percent change is handled that resulted in the progress bar not moving in response to such messages, but that wasn't the best fix. Thanks for pointing that out.